IFC4x3/Properties/f/FaceInclination_0KcNnRVov7v8l33uxXOfnq/Documentation.md

```diff
@@ -0,0 +1 @@
+Inclination of face inclination.
```

Comment:
Inclination of face inclination? 😄 

Should this have been "Excavation face inclination."?

Reply:
This comment should be answered by @JontyDunn

Reply:
Name changed to ExcavationFaceInclination (including PEnum_ExcavationFaceInclination).